$MirOS: ports/plan9/kencc/patches/patch-utils_qa_a_h,v 1.1 2006/03/31 03:50:23 tg Exp $
--- utils/qa/a.h.orig	Thu Mar 30 23:54:38 2006
+++ utils/qa/a.h	Thu Mar 30 23:55:25 2006
@@ -137,7 +137,7 @@ EXTERN	char*	thestring;
 EXTERN	long	thunk;
 EXTERN	Biobuf	obuf;
 
-void	errorexit(void);
+void	errorexit(void) __attribute__((__noreturn__));
 void	pushio(void);
 void	newio(void);
 void	newfile(char*, int);
